### DISCUSSION TITLE
input data format

### DISCUSSION THREAD

**Discussion by INFINITY1993** [antismash/antismash#428] [Q&A] [answered]
Hi, 
I am going to predict BGCs in metagenome-assembled genomes (MAGs) recovered from environmental sample. How could I convert the  fasta files into gbk format? Is antiSMASH applicable to predict BGCs in contigs?
Thanks!

**Comment by SJShaw** [comment, marked as answer]
You'll need some genefinding to get a meaningful genbank from a fasta file. Software like prokka has streamlined some of that.
You can get antiSMASH to do genefinding, but you can generally outperform the base prodigal genefinding in antiSMASH if you can customise the process.

The main drawback to metagenomics in antiSMASH is that the clusters are very often interrupted by contig edges, which prevent them from being detected.

**Reply by INFINITY1993** [reply]
Thanks so much for your responses.
